### PR TITLE
changes for DXCB, eslint, package to run prettier, ignore index.html

### PR DESCRIPTION
**Pull request by paulgagnon-pega**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,10 +125,10 @@
     "@angular/language-service": "^16.2.12",
     "@pega/angular-sdk-overrides": "file:pega-angular-sdk-overrides-8.23.10.tgz",
     "@pega/configs": "^0.7.1",
-    "@pega/dx-component-builder-sdk": "~0.1.49",
+    "@pega/dx-component-builder-sdk": "file:pega-dx-component-builder-sdk-23.1.12.tgz",
```
Comment (vishalshrm539) [inline package.json:128]: Local tgz reference should be removed

File: package.json [hunk 2]
```diff
@@ -13,7 +13,7 @@
     "fix:es": "eslint --color --fix --cache --cache-location node_modules/.cache/eslint/ src",
     "fix:format": "prettier --log-level warn -w .",
     "start-dev": "http-server ./dist --port 3500 --cors --proxy http://localhost:3500?",
-    "start-dev-https": "http-server ./dist --ssl --cert private.pem --key private.key --port 3500 --cors --proxy https://localhost:3500?",
+    "start-dev-https": "http-server ./dist --ssl --cert /Users/gagnp/keys/private.pem --key /Users/gagnp/keys/private.key --port 3500 --cors --proxy https://localhost:3500?",
```
Comment (vishalshrm539) [inline package.json:16]: Shouldn't we get rid of `Users/gagnp/` here?

File: package.json (hunk 1 above)
Comment (paulgagnon-pega) [reply package.json:128]: for now done intentionally, we can generate a tgz file for it.  I can generate one and put in sharepoint, but will only have my 5 template changes to SDK-Angular part.  Torque, can pull the code from bitbucket and generate a tgz file (will have same name) and be able to move forward.  Think for now, like the other tgz files, it is the best, so we don't have to keep updating the package json.

Comment (paulgagnon-pega) [reply package.json:128]: dxcb sdk tgz with latest https://pegasystems.sharepoint.com/:u:/s/SP-DXInnovationLab/ESpIl1HyZHtBjslhrf-LbroBrNkrA_L_WcUd2arC9hHegg?e=s3vmXI

File: package.json (hunk 2 above)
Comment (paulgagnon-pega) [reply package.json:16]: sorry, I will get rid of that

**Review by vishalshrm539 — APPROVED** [review]

**Pull request merged**